src/batGc.mliv

```diff
@@ -227,7 +228,8 @@ val allocated_bytes : unit -> float
     started.  It is returned as a [float] to avoid overflow problems
     with [int] on 32-bit machines. *)
 
-##V>=4.3##external get_minor_free : unit -> int = "caml_get_minor_free" [@@noalloc]
+##V>=4.3##external get_minor_free : unit -> int = "caml_get_minor_free"
+##V<4.5##          [@@noalloc]
```

Comment:
Here we need to have `[@@noalloc]` show up under both 4.03 and 4.04, but no other versions. Unfortunately, our prefilter code does not support disjunctions, so you may have to duplicate the line:

```
##V=4.3## [@@noalloc]
##V=4.4## [@@noalloc]
```